spec/models/project_spec.rb

```diff
@@ -7,4 +7,44 @@
     it {should have_many :contestant_projects}
     it {should have_many(:contestants).through(:contestant_projects)}
   end
+
+  describe "#contestant_count" do
+    it 'calculates the count of all contestants' do
+      furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+
+      lit_fit = furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+
+      jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+      gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+      kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+
+      # lit_fit.contestants << [jay, gretchen, kentaro]
+
+      jay.projects << lit_fit
+      gretchen.projects << lit_fit
+      kentaro.projects << lit_fit
+
+      expect(lit_fit.contestant_count).to eq(3)
```

Comment:
With model method tests it's totally fine to just expect an integer here though, like normal! No concern for false positives in this case. 

Awesome job testing this functionality at both the unit (model spec) and integration (feature spec) level!